python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -536,45 +543,57 @@ def _internal_job_finalisation(job, updated_dirac_status):
         logger = getLogger()
 
         if updated_dirac_status == 'completed':
+            import time
+            start = time.time()
             # firstly update job to completing
             DiracBase._getStateTime(job, 'completing')
             if job.status in ['removed', 'killed']:
                 return
-            if (job.master and job.master.status in ['removed', 'killed']):
+            elif (job.master and job.master.status in ['removed', 'killed']):
```

Comment:
No, no harm. I find it *slightly* harder to parse but not much. I was just wondering if there was a reason for the change.